src/Upgrade/Infrastructure/PackageManager/Reader/ComposerLockReader.php

```diff
@@ -13,6 +13,16 @@
 
 class ComposerLockReader implements ComposerLockReaderInterface
 {
+    /**
+     * @var int
+     */
+    protected int $modifyTime = 0;
```

Comment:
Why do we need this property? It is used only in one place and will always be 0. Can be removed, imo.